spec/requests/landing_pages_spec.rb

```diff
@@ -0,0 +1,245 @@
+require "spec_helper"
+
+def create_version(community_id, structure, version_number)
+  s = structure.deep_dup
+  s["page"]["title"] = {value: "Title version #{version_number}"}
+  FactoryGirl.create(
+    :landing_page_version,
+    community_id: community_id,
+    version: version_number,
+    content: JSON.generate(s)
+  )
```

Comment:
I'm not 100% factory girl is necessary / a good idea here. The other option would be to extend CustomLandingPage::LandingPageStore to include methods for writing data. It would then fully encapsulate the data access and make ActiveRecord an implementation detail. That's why I didn't originally add the model objects under models.